.github/test-spec.yml

```diff
@@ -37,10 +37,6 @@
   - "mpsl/lib/**/*"
 
 "CI-mesh-test":
-  - "softdevice_controller/include/**/*"
-  - "softdevice_controller/lib/**/*"
-  - "mpsl/include/**/*"
-  - "mpsl/lib/**/*"
```

Comment:
is leaving empty entries accepted?

Reply:
according to yml spec empty should get convert to null safely.  
but you could add it to be safe. 
`"CI-mesh-test": null`